.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  workflow_dispatch:
+
+jobs:
+  lint:
+    name: "Lint & Packaging"
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: "actions/setup-python@v2"
+        with:
+          python-version: "3.6"
+
+      - name: "Install dependencies"
+        run: |
+          set -xe
+          python -VV
+          python -m site
+          python -m pip install --upgrade pip setuptools wheel
+          python -m pip install --upgrade tox
+      - name: "Run tox lint targets for ${{ matrix.python-version }}"
+        run: "python -m tox -e check_isort,pep8,packaging"
```

Comment:
How come we do `python -m tox` here and just `tox` below?

Reply:
I don't think there's a particular reason. Probably because I copied these lines out of different configs.